### ISSUE TITLE
Missing current line highlight background

### ISSUE THREAD

**Issue by akaRem** [dracula/visual-studio-code#68]
According to screen from README, current line should be highlighted with #44475a or similar colour.
Reference from readme:
<img  src="https://camo.githubusercontent.com/d695efd1815607a9056ea5817501adc0a63632ca/68747470733a2f2f64726163756c617468656d652e636f6d2f6173736574732f696d672f73637265656e73686f74732f7673636f64652e706e67">

But in my installation I see only border.
Theme: 2.9.0
VSCode: 1.21.1
macOS: 10.12.6

My version:
<img width="360" alt="2018-03-18 0 42 42" src="https://user-images.githubusercontent.com/1472728/37560205-4bd024de-2a45-11e8-8d35-e46c0adb3a79.png">

I fixed it with config as follows
```
    "workbench.colorCustomizations": {
        "editor.lineHighlightBackground": "#44475a"
    }
```
And now it looks like this
<img width="428" alt="2018-03-18 0 46 32" src="https://user-images.githubusercontent.com/1472728/37560247-d95e7c88-2a45-11e8-9456-a340dfe0b6c5.png">

I'm not sure how it should look to match theme guideline, but from my point of view fixed version matches better.

I'm not experienced w/ code themes, but it looks like the problem is somewhere in this line:
https://github.com/dracula/visual-studio-code/blob/master/src/dracula.yml#L184

Could you please review proposed change and fix theme if you agree on this proposition?

**Comment by dsifford** [comment]
Thanks for the report.

The image in the readme is not correct. The line highlight background is purposely not set due to how VSCode deals with overlapping highlights.

Instead of using the topmost highlight color for each individual block in the editor, VSCode instead overlays the highlight colors on top of each other. 

This becomes a problem in situations where you have several highlights being applied at once. For example, consider a situation where you have line highlight enabled (1) and you are in the diff viewer on an active diff (2), with a search matching the area you are on (3), with an overlapping part of that area also selected (4).

That situation results in the following overlaps:

```txt
4 | text selection
3 | search highlight
2 | diff highlight
1 | line highlight
```

The opacities of the highlights are merged and the line will likely become nearly fully opaque.

So, in an effort to cut down on some of the overlapping noise of the line highlight, that's the reason why we have it disabled.

As you've shown, you are encouraged to enable that setting if you feel like that is a better look for your individual needs.

**TL;DR:** This is by design.

**Comment by jxramos** [comment]
Is there a way to control the style in some sort of hierarchy/precedence of rules much like css conducts?
https://stackoverflow.com/a/25105841/1330381

Those rules get pretty complex, not sure if there's an elegant way to tease out those overlaps from VSCode but would be a nice feature I'd imagine.
